### PR TITLE
Implemented View Profile

### PR DESCRIPTION
**Pull request by JoeyPL5**
Added route for other users profiles 
Added route for user does not exist page
Created other user's profile page
Created user does not exist page

### CODE REVIEW

File: src/home/Home.js [hunk 1]
```diff
@@ -148,6 +150,8 @@ export default function Home() {
         <Route path="/admin" element={<Admin />} />
         <Route path="/details/:id" element={<Details />} />
         <Route path="/myprofile" element={<MyProfile />} />
+        <Route path="/profile/:username" element={<OtherProfile />} />
+        <Route path="/profile/dne" element={<UserDNE />} />
```
Comment (andrewli5) [inline src/home/Home.js:154]: switch this to /404 or something cuz what if a user is named dne

File: src/profile/UserDNE.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Typography } from "@mui/material";
+const UserDNE = () => {
+    return (
+        <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '20vh' }}>
+            <div>
+                <Typography variant="h1" style={{ color: 'gray' }}>User Does Not Exist</Typography>
```
Comment (andrewli5) [inline src/profile/UserDNE.js]: make the text all lowercase

**Comment by andrewli5** [comment]
also get rid of the mongoose import in Search.js i think i autocompleted that by mistake oops thats why the deploy fails

File: src/home/Home.js (hunk 1 above)
Comment (JoeyPL5) [reply src/home/Home.js:154]: what if a user is named 404
is there any characters they cant use or something

Comment (andrewli5) [reply src/home/Home.js:154]: no like /404 not /profile/404

**Pull request merged**